### ISSUE TITLE
04_model_architecture notebook completion

### ISSUE THREAD

**Issue by bajon1** [bajon1/Classification-pipeline-on-the-Mental-Health#5]
- implementation of the MLP class (imported from src/model.py)
- demonstration of the architecture
- train_epoch() and validate_epoch() functions with gradient clipping
- training loop for N epochs using SummaryWriter.

**Comment by bajon1** [comment]
b974ba1395bd5bfec3965719cea567f71e63da58
